source/utilities.tex

```diff
@@ -2664,6 +2910,82 @@
 result of the comparison.
 \end{note}
 
+\rSec2[tuple.common.ref]{\tcode{common_reference} specialization}
+
+\pnum
+In the descriptions that follow:
+\begin{itemize}
+\item
+Let \tcode{TTypes} be a pack formed by
+the sequence of \tcode{tuple_element_t<$i$, TTuple>}
+for every integer $0 \leq i < \tcode{tuple_size_v<TTuple>}$.
+
+\item
+Let \tcode{UTypes} be a pack formed by
+the sequence of \tcode{tuple_element_t<$i$, UTuple>}
```

Comment:
Is it OK to not explicitly say "ordered sequence", like in [tuple.cat]?

Reply:
? not an edit issue.